content/en/docs/getting-started/packages/_index.md

```diff
@@ -67,21 +67,33 @@ To access the service logs, use the following command:
 $ sudo journalctl -fu iam-login-service
 ```
 
-## Automated provisioning with Puppet
+### Deployment Tips
+In headless servers, running `haveged` daemon is recommended to generate more entropy.
```

Comment:
Cross-checked that it is installed also on our k8s cluster where IAM runs, _e.g._
```
[fagostini@kube-master ~]$ sudo yum list installed | grep haveged
haveged.x86_64                         1.9.13-1.el7                    @epel

[fagostini@kube-node07 ~]$ sudo yum list installed | grep haveged
haveged.x86_64                              1.9.13-1.el7               @epel
```